web-ui/src/main/resources/catalog/components/edit/multientrycombiner/partials/multientrycombiner.html

```diff
@@ -0,0 +1,49 @@
+<div>
+
+  <!--- These are the actual values returned to the server -->
+  <input type="hidden" ng-repeat="(lang,text) in config.values"  value="{{text}}" name="_lang_{{lang}}_{{root_id}}" lang="{{lang}}"/>
+
+
+<div>
+  <div class="form-group" ng-repeat="c in config.config" ng-if="c.type !== 'fixedValue'">
+    <div>
+      <label for="field_{{$index}}" class=" ">{{c.heading[currentLang]}}</label>
+    </div>
+
+    <div ng-if="c.type == 'fixedValue'">
```

Comment:
I think this can be removed, due to the `ng-if` in line 8?

Reply:
done